### PR TITLE
JPN-453 Community Page theming

### PR DESCRIPTION
**Pull request by lukaszkonieczny**
Moving from https://github.com/Wikia/app/pull/10417 while JPN-407 is already merged.

Started as a part of refactoring suggested in https://github.com/Wikia/app/pull/10399#discussion_r62391451 went a bit broader and added support for custom wiki colors.

The actual theming ticket is here:
https://wikia-inc.atlassian.net/browse/JPN-453

@Wikia/spitfires @perjg @d34th4ck3r

### CODE REVIEW

File: extensions/wikia/CommunityPage/styles/components/_header.scss [hunk 1]
```diff
@@ -40,7 +40,7 @@
 
 .community-page-admin-welcome-message {
 	align-items: center;
-	background-color: $color-community-page-gray;
+	background-color: $color-unified-module-background;
 	color: $color-text-dark;
```
Comment (lukaszkonieczny) [inline extensions/wikia/CommunityPage/styles/components/_header.scss]: `background-color` is calculated based on wiki theme so text with `$color-text-dark` may not has enough contrast.

File: extensions/wikia/CommunityPage/styles/components/_modal.scss [hunk 2]
```diff
@@ -19,28 +19,28 @@
 		}
 
 		a {
-			color: $color-nav-inactive-text;
+			color: $color-alternate-text;
```
Comment (lukaszkonieczny) [inline extensions/wikia/CommunityPage/styles/components/_modal.scss:25]: Why not `$color-links` ?

File: extensions/wikia/CommunityPage/styles/components/_variables.scss [hunk 3]
```diff
@@ -1,32 +1,15 @@
 @import 'skins/shared/color';
 
 // colors
-$color-community-page-white: #fff;
-$color-community-page-gray: #eee;
-$color-community-page-light-gray: #e6e6e6;
-$color-community-page-dark-gray: #ccc;
 $color-community-page-admin-text: #808080;
```
Comment (lukaszkonieczny) [inline extensions/wikia/CommunityPage/styles/components/_variables.scss]: I cannot find usage of this.

File: extensions/wikia/CommunityPage/styles/components/_variables.scss [hunk 4]
```diff
@@ -1,32 +1,15 @@
...
 
 $color-activity-background: #f6f6f6;
```
Comment (lukaszkonieczny) [inline extensions/wikia/CommunityPage/styles/components/_variables.scss]: I cannot find usage of this.

File: extensions/wikia/CommunityPage/styles/CommunityPage.scss [hunk 5]
```diff
@@ -2,7 +2,6 @@
 @import 'skins/shared/mixins/transform';
```
Comment (lukaszkonieczny) [inline extensions/wikia/CommunityPage/styles/CommunityPage.scss]: I think this is not needed

File: extensions/wikia/CommunityPage/styles/components/_header.scss (hunk 1 above)
Comment (kamilkoterba) [inline extensions/wikia/CommunityPage/styles/components/_header.scss]: Sorry, we were aware but didn't make it with @d34th4ck3r before CR ;)

File: extensions/wikia/CommunityPage/styles/components/_modal.scss (hunk 2 above)
Comment (kamilkoterba) [inline extensions/wikia/CommunityPage/styles/components/_modal.scss:25]: According to designs provided active tab that has link color and other tabs are grayed.

**Comment by lukaszkonieczny** [comment]
LGTM

**Comment by adamkarminski** [comment]
Testing LGTM, merging

**Pull request merged**